en/core-api/credit-card.md

```diff
@@ -246,13 +233,12 @@ $response = \Midtrans\CoreApi::charge($params);
 ```
 
 #### **Node JS**
+For Node JS, install [midtrans-client NPM package](https://github.com/Midtrans/midtrans-nodejs-client).
 
-Install [**midtrans-client**](https://github.com/Midtrans/midtrans-nodejs-client) NPM package
-```bash
-npm install --save midtrans-client
-```
+`npm install --save midtrans-client`
```

Comment:
Why is this formatting get changed? This become inconsistent with the Python tabs, because you didn't change the Python tabs, but changed this

Reply:
Resolved